js/api/item/dummy-data.js

```diff
@@ -1,5 +1,3 @@
-/* global require */
```

Comment:
This change should be out of this pr, but we have seen others things in this repo right ? ;)